lib/ansible/modules/clustering/consul.py

```diff
@@ -277,10 +279,12 @@ def add(module):
 
 def remove(module):
     ''' removes a service or a check '''
-    service_id = module.params.get('service_id') or module.params.get('service_name')
+    service_id = module.params.get(
+        'service_id') or module.params.get('service_name')
```

Comment:
I'd avoid reformatting the code in a pr like this, formatting is pretty subjective and this is not related to your change. If you're adamant about the formatting please submit a separate pr.